### PR TITLE
chore: Collect LAM service config in support bundle

### PR DESCRIPTION
**Pull request by banjoh**
#### What this PR does / why we need it:

Collect Local Artefact Mirror systemd service config in support bundle

#### Does this PR require a test?
NONE

#### Does this PR require a release note?
```release-note
Collect local artefact mirror service config in support bundle
```

#### Does this PR require documentation?
NONE

### CODE REVIEW

File: pkg/goods/support/host-support-bundle.tmpl.yaml [hunk 1]
```diff
@@ -153,6 +153,9 @@ spec:
       collectorName: 'check-umount'
       command: 'sh'
       args: ['-c', 'command -v umount']
+  - copy:
+      collectorName: installer/lam-service-config
+      path: /etc/systemd/system/local-artifact-mirror.service.d/embedded-cluster.conf
```
Comment (emosbaugh) [inline pkg/goods/support/host-support-bundle.tmpl.yaml:158]: does this support globs? is this possible?

```suggestion
      path: /etc/systemd/system/local-artifact-mirror.service.d/*
```

File: operator/charts/embedded-cluster-operator/templates/embedded-cluster-lam-service-config.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    spec:
+      collectors:
+        - runDaemonSet:
+            name: "local-artifact-mirror-service-config"
+            namespace: embedded-cluster
+            podSpec:
+              containers:
+              - image: {{ .Values.utilsImage }}
+                imagePullPolicy: Always
+                args: ["chroot","/host","cat","/etc/systemd/system/local-artifact-mirror.service.d/embedded-cluster.conf"]
```
Comment (emosbaugh) [inline operator/charts/embedded-cluster-operator/templates/embedded-cluster-lam-service-config.yaml:27]: would it be possible to collect all files in this directory?

File: pkg/goods/support/host-support-bundle.tmpl.yaml (hunk 1 above)
Comment (laverya) [reply pkg/goods/support/host-support-bundle.tmpl.yaml:158]: I think just a `/` is enough, no need for the `/*`

Comment (laverya) [reply pkg/goods/support/host-support-bundle.tmpl.yaml:158]: https://troubleshoot.sh/docs/host-collect-analyze/copy/ `*` is in the logs though, so yeah let's go with that

File: operator/charts/embedded-cluster-operator/templates/embedded-cluster-lam-service-config.yaml (hunk 2 above)
Comment (banjoh) [reply operator/charts/embedded-cluster-operator/templates/embedded-cluster-lam-service-config.yaml:27]: Something like `"/host","sh", "-c", "cat /etc/systemd/system/local-artifact-mirror.service.d/*"` will work. All files would be concatenated into one file. That might be confusing. I think its better to collect one file at a time

**Review by emosbaugh — APPROVED** [review]

**Pull request merged**